monkey/agent_plugins/exploiters/hadoop/config-schema.json

```diff
@@ -0,0 +1,43 @@
+{
+    "title": "Hadoop/Yarn Exploiter",
+    "description": "Remote code execution on HADOOP server with YARN and default settings. Logic based on https://github.com/vulhub/vulhub/tree/master/hadoop/unauthorized-yarn.",
+    "required": [
+        "exploitable_ports",
+        "http_download_timeout",
+        "app_random_string_length"
+    ],
+    "properties": {
+        "exploitable_ports": {
+            "title": "List of exploitable ports",
+            "description": "A list of potential exploitable ports that Hadoop will try to exploit.",
```

Comment:
We could use @ybasford's help with these titles and descriptions.

Reply:
Title should probably be "Target ports" or "Destination ports". I'd say we already have "target" in some place, so let's use that.
Description could be jus "A list of ports that Hadoop exploiter will try to exploit."
Feel free to run this through @ybasford